src/login/71-seat.rules.in

```diff
@@ -24,6 +24,9 @@ SUBSYSTEM=="usb", ATTR{idVendor}=="2230", ATTR{idProduct}=="000[13]", ENV{ID_AUT
 #     http://git.qemu.org/?p=qemu.git;a=blob;f=docs/multiseat.txt
 SUBSYSTEM=="pci", ATTR{vendor}=="0x1b36", ATTR{device}=="0x000a", TAG+="seat", ENV{ID_AUTOSEAT}="1"
 
+# Video adapter of Parallels virtualization platform
+SUBSYSTEM=="pci", ATTRS{vendor}=="0x1ab8", ATTRS{device}=="0x4005", TAG+="seat", TAG+="master-of-seat"
```

Comment:
why not set ID_AUTOSEAT here?

Reply:
If I get it right this doesn't work: setting ID_AUTOSEAT creates new seat for this video adapter but input devices etc. go to default seat0 (without any video adapter).

Reply:
@poettering could you please take a look? Do you still have any objections?